sdk/eventhub/Azure.Messaging.EventHubs/src/Retry.cs

```diff
@@ -50,7 +50,7 @@ public abstract class Retry
         ///
         /// <returns><c>true</c> if the operation that produced the exception may be retried; otherwise, <c>false</c>.</returns>
         ///
-        internal static bool IsRetriableException(Exception exception) => throw new NotImplementedException();
+        internal virtual bool IsRetriableException(Exception exception) => throw new NotImplementedException();
```

Comment:
On a tangent, in Java, they use "retryable", made me look more into it. Lol.
https://english.stackexchange.com/questions/305273/retriable-or-retryable